### DISCUSSION TITLE
Is there a way to make "close siblings"?

### DISCUSSION THREAD

**Discussion by ooker777** [SkepticMystic/breadcrumbs#309] [Q&A]
I am making an [issue tree](https://en.wikipedia.org/wiki/Issue_tree "Issue tree - Wikipedia"), and I want to add the logical operator `AND` and `OR` for the solutions of the issues. For example, for an issue, you may solve it with solution 1a *and* solution 1b, or you can solve it with solution 2. So while they are siblings, 1a and 1b are close siblings. Is there a way to represent this relationship?

**Comment by michaelpporter** [comment]
In the v4 rewrite this is doable by defining your **own edge field** for the tight grouping, separate from the normal sibling field.

Regular siblings use the built-in `same` field. For your "close siblings" (the AND-group — 1a *and* 1b), create a dedicated field, e.g. `and` (or `co-sibling`), under **Settings → Breadcrumbs → Edge fields**. Then in your notes:

```yaml
---
same: ["[[Solution 2]]"]      # loose sibling (OR alternative)
and: ["[[Solution 1b]]"]       # close sibling (AND-group)
---
```

If you want it to be symmetric (so 1b also points back to 1a automatically), add a transitive rule `[and] -> and` under **Implied relations** — same pattern the default `same` field uses.

That gives you two distinct sibling relationships you can style and traverse independently. Closing as resolved — v4 makes this a matter of adding a custom field; shout if you hit a snag.
